admin-dev/themes/new-theme/template/light_display_layout.tpl

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="{$iso}">
+<head>
+  {$header}
+</head>
+
+<body
+  class="lang-{$iso_user}{if $lang_is_rtl} lang-rtl{/if} {$smarty.get.controller|escape|strtolower}{if $collapse_menu} page-sidebar-closed{/if}"
+  {if isset($js_router_metadata.base_url)}data-base-url="{$js_router_metadata.base_url}"{/if}
+  {if isset($js_router_metadata.token)}data-token="{$js_router_metadata.token}"{/if}
+>
+
+<div id="main-div" class="light_display_layout">
+    {if $install_dir_exists}
+      <div class="alert alert-warning">
+        {l|escape s='For security reasons, you must also delete the /install folder.' d='Admin.Login.Notification'}
+      </div>
+    {else}
+      {if isset($modal_module_list)}{$modal_module_list}{/if}
+
+      <div class="-notoolbar">
+        {hook h='displayAdminAfterHeader'}
+
+        {if $display_header}
+          {include file='components/layout/error_messages.tpl'}
+          {include file='components/layout/information_messages.tpl'}
+          {include file='components/layout/confirmation_messages.tpl'}
+          {include file='components/layout/warning_messages.tpl'}
+        {/if}
+
+        {$page}
+        {hook h='displayAdminEndContent'}
+      </div>
+    {/if}
+</div>
+
+{if isset($php_errors)}
+  {include file="error.tpl"}
+{/if}
+</body>
+</html>
```

Comment:
We could add some padding inside the html tag honestly, but I'm fine with that new layout system for modals, I think we've some modals with iframe

Reply:
Yeah I agree but it means each time we use a controller in a modal we need custom CSS rules to prevent this bug So I was aiming for a more global solution

Of course instead of customizing CSS we need to customize the layout rendering instead so it's not fully automatic, but at least we now have a cleaner layout for light display And we'll be able to reuse it in a future refacto when we clean the way `liteDisplay` is handled